packages/lmi/src/lmi/rate_limiter.py

```diff
@@ -329,6 +329,13 @@ async def rate_limit_status(self):
             }
         return limit_status
 
+    def maybe_lower_weight_with_rate_resolution(
+        self, weight: int, rate_limit: RateLimitItem
+    ) -> int:
+        if hasattr(rate_limit, "resolution"):
+            return max(1, weight // rate_limit.resolution)
+        return weight
```

Comment:
Do you mind moving this to a method of `RateLimitItem`? We aren't using `self` here, so it doesn't need to be on `GlobalRateLimiter`

The below would become:

```python
        adjusted_weight = rate_limit.maybe_lower_weight_via_resolution(weight)
```

Which feels cleaner

Reply:
Oh if it's not a custom class of ours, don't subclass for this. Just do a `staticmethod` or free function here

Or just totally ignore this comment, lol